openpype/hosts/maya/lib.py

```diff
@@ -1,26 +1,23 @@
 import os
-import shutil
+from openpype.settings import get_project_settings
+from openpype.api import Logger
```

Comment:
With refactor `openpype.api` begins to be a relic, use more specific imports.
```suggestion
from openpype.lib import Logger
```